docs/guides/references/troubleshooting.mdx

```diff
@@ -247,34 +247,69 @@ an issue you have by clearing this app data.
 
 ## Print DEBUG logs
 
-Cypress is built using the [debug](https://github.com/visionmedia/debug) module.
-That means you can receive helpful debugging output by running Cypress with this
-turned on. **Note:** you will see a LOT of messages when running with
+Cypress is built using the [debug](https://github.com/debug-js/debug) module.
+That means you can receive helpful debugging output by running Cypress with
+the environment variable `DEBUG` set to `cypress:*` prior to running `cypress run` or `cypress open`.
+**Note:** you will see a LOT of messages when running with the
 `DEBUG=...` setting.
 
-**On Mac or Linux:**
+**On macOS, Linux or Windows (Git Bash):**
```

Comment:
I would say that capturing logs in docker or CI is sometimes a challenge for people - not sure if you want to link to the other docs on that in docker-images / github actions, etc.

Reply:
Good suggestion. I'll make a note to tackle this subject in a follow-on PR.